bosh-director/lib/bosh/director/jobs/ssh.rb

```diff
@@ -38,8 +38,9 @@ def perform
             if target.ids_provided?
               result["id"] = instance.uuid
             else
-              result["index"] = instance.index
+              result["index"] = result["id"] = instance.index
```

Comment:
Should we always send back `id` and `index`? Including both values seems valid for an SSH results response. Then the UI/cli becomes responsible for showing the user whichever is appropriate. Or maybe that would impact the ruby CLI in ways we want to avoid? Dunno; just a thought.